expression_data_matrix.md

```diff
@@ -40,6 +40,43 @@ The bundle metadata is intended to contain metadata common to all the samples an
 
 It may occur that the matrix contains samples from multiple projects or studies.  If elements like the processing pipeline are no longer common to all samples, care should be taken to move this metadata to the samples or features metadata table instead of the bundle metadata.
 
+# Sample Metadata Discussion
+
+### Minimum Sample Metadata
+
+The following metadata fields describe a recommended minimal set of sample metadata.
+
+<table>
+<tr markdown="block"><td>
+`sampleName`
+</td><td>
+_string_
+</td><td>
+A human readable name for the sample.  Would be better if it were unique to the sample within the database.
+</td></tr>
+<tr markdown="block"><td>
+`tissue`
```

Comment:
`tissue` might be inappropriate in some cases? (e.g. immortalised cell lines)

Reply:
good catch - I'm making an update to use the MIABIS materialType which covers tissue, blood, cell lines, etc.